README.md

```diff
@@ -219,7 +222,40 @@ class { 'sensu::backend':
 }
 ```
 
-The type `sensu_ldap_auth` requires a valid enterprise license.
+The types `sensu_ad_auth` and `sensu_ldap_auth` require a valid enterprise license.
+
+### PostgreSQL datastore support
+
+**NOTE**: This features require a valid Sensu Go enterprise license.
+
+The following example will add a PostgreSQL server and database to the sensu-backend host and configure Sensu Go to use PostgreSQL as the event datastore.
+
+```puppet
+class { 'postgresql::globals':
+  manage_package_repo => true,
+  version             => '9.6',
+}
+class { 'postgresql::server': }
```

Comment:
Is there a reason to use `class{}` with no params instead of `include` ?

Reply:
Because that's what puppetlabs documents. Also ensuring we don't do something to break the ability for people to use `class { } ` since that's easier to break than just `include`.